__tests__/unit/gitlab/MilestoneService.test.ts

```
@@ -0,0 +1,92 @@
+import { GitLabClient } from '../../../app/api/gitlab/client/GitLabClient';
+import { ConsoleLogger } from '../../../app/api/gitlab/client/Logger';
+import { MilestoneService } from '../../../app/api/gitlab/services/MilestoneService';
+import { GitLabConfig, MilestoneState } from '../../../app/api/gitlab/types';
```

ESLint

[error] 'GitLabConfig' is defined but never used.

(@typescript-eslint/no-unused-vars)